packages/flutter_markdown/test/table_test.dart

```diff
@@ -441,8 +462,7 @@ void defineTests() {
               .toList();
           expect(text[0], '| abc | def | | --- | | bar |');
         },
-        // TODO(mjordan56): Remove skip once the issue #341 in the markdown package
-        // is fixed and released. https://github.com/dart-lang/markdown/issues/341
+        // TODO(kevmoo): Remove skip once min pkg:markdown > 6.0.1
```

Comment:
A win here!

Reply:
Any reason not to bump the minimum Markdown version now and clean this up as opposed to leaving this tech debt in the code base?